### ISSUE TITLE
Mortal Kombat II + bug

### ISSUE THREAD

**Issue by Zoltan45** [libretro/mame2003-plus-libretro#1864]
If you start a game of 2v2 using this core, the first time you attack an opponent, they die and the second character comes out.  This is not the expected behavior.

This attempt was made on a Windows 10 PC, Retroarch 64 bit 1.19.1 and an updated core.

**Comment by mahoneyt944** [comment]
Is this bug unique to this core? Does the same issue occur for say FBNEO?

**Comment by arcadez2003** [comment]
It could be a bug in the pcb which was originally dumped which was fixed in a later revision
try anther Mortal Kombat II set and see if the issue persists.

**Comment by Zoltan45** [comment]
> Is this bug unique to this core? Does the same issue occur for say FBNEO?

no, it will also occur on mame2003, and likely anything that age or older.  This is a known mame bug with the game that i have somehow managed to glance over for years now.  I believe this is why the original faqs for the game stated you must use mame 119 or newer.  I don't think that's necessarily the point at which it was fixed, but the oldest mame the devs worked with that worked. So the issue was fixed somewhere between mame 0.78 and mame 0.119.

**Comment by Zoltan45** [comment]
> What about MAME2003 does the issue happen using that core.??
> 
> But It could be a bug in the pcb which was originally dumped which was fixed in a later revision try anther Mortal Kombat II set and see if the issue persists.

Yes.

I am not aware of this game being dumped at all.

Its a hack that you patch and burn the roms for your board or put in to mame to play.   In any case, this is already the newest revision of the game and it works fine on newer cores and standalone emulators.

**Comment by arcadez2003** [comment]
Ah right didn't spot the + in the title so we're talking about the hack then not the original sets
@mahoneyt944 as per this fix maybe the issue is CPU related when it comes to these hacks...
https://github.com/libretro/mame2003-plus-libretro/commit/ff076e6982e3ecf23cc9fa2bcff5679fb024092d

**Comment by mahoneyt944** [comment]
I recall that fix being required to fix umk3p ( jade reset glitch ).  It's hard to say what would fix this issue for mk2p without plugging in every change and testing.

**Comment by Zoltan45** [comment]
> I recall that fix being required to fix umk3p ( jade reset glitch ). It's hard to say what would fix this issue for mk2p without plugging in every change and testing.

I think I have enough time today to try and track down when it was fixed in standalone mame at the least.

**Comment by mahoneyt944** [comment]
That would help a lot

**Comment by Zoltan45** [comment]
This bug is not present in MAME 115 but is still present in MAME 114.

Theres a lot of TMS code change between.  but at least its pinned down if anyone gets time and wants to look at it (self included).
